pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -222,6 +240,44 @@ func (f *framework) RunPrefilterPlugins(
 	return nil
 }
 
+// RunScorePlugins runs the set of configured scoring plugins. It returns a map that
+// stores for each scoring plugin name the corresponding NodeScoreList(s).
+// It also returns *Status, which is set to non-success if any of the plugins returns
+// a non-success status.
+func (f *framework) RunScorePlugins(pc *PluginContext, pod *v1.Pod, nodes []*v1.Node) (PluginToNodeScoreMap, *Status) {
+	pluginToNodeScoreMap := make(PluginToNodeScoreMap, len(f.scorePlugins))
+	for _, pl := range f.scorePlugins {
+		var (
+			mu      = sync.Mutex{}
```

Comment:
there is no need to pass a default value to a mutex when initialize

```go
var (
mu sync.Mutex
```

https://play.golang.org/p/pOeTd2c2tOo